### ISSUE TITLE
JDK10 in travis build

### ISSUE THREAD

**Issue by pshirshov** [7mind/izumi#150]
Blocked by https://github.com/travis-ci/travis-ci/issues/9368

**Comment by pshirshov** [comment]
Waiting for scala 2.12.7, in 2.12.6 the optimizer is broken.

**Comment by pshirshov** [comment]
Already done.
